bin/dev/fake-git

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I don't think this should be covered. On the other hand, we do use `"$@"`, and that _might_ be a bashism.

We should prefer `/usr/bin/env bash`, though (on account of some machines having it in `/usr/bin/bash`).